js/application.js

```diff
@@ -1 +1,19 @@
 // instantiate a StockView
+var field = document.getElementById("ticker");
+var button = document.getElementById("lookup-stock");
+var sharesField = document.getElementById("shares-field");
+var sharesButton = document.getElementById("enter-shares");
+
+var stock = new Stock();
+
+var stockView = new StockView(stock);
+
+button.addEventListener("click", function(){
+	stock.updateStock(field.value);
+	stockView.render();
+})
+
+sharesButton.addEventListener("click", function(){
+	stock.updateShares(sharesField.value);
+	stockView.render();
+})
```

Comment:
so these `addEventListener`'s are in the global namespace.  Which, if you can imagine, on a much larger scale application could turn into jQuery soup. If we were to move these into the view definition. then we could do something like `this.stock.update(field.value)` or `this.render()` Ideally, you want to move these event listeners into the view for an OOJS solution.